custom_components/fritzbox_vpn/config_flow.py

```
@@ -96,7 +118,7 @@ async def validate_input(hass: HomeAssistant, data: Dict[str, Any]) -> Dict[str,
             )
             raise InvalidAuth from err
         _LOGGER.exception("Error validating input: %s", err)
```

Ruff (0.15.1)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)